src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/settings/View.tsx

```diff
@@ -85,11 +85,10 @@ export const SettingsView = (props: Props) => {
           <EmailAddressAdder />
           <hr />
           <AlertAddressForm
-            defaultSelected={
-              props.user.subscriber?.all_emails_to_primary
-                ? "primary"
-                : "affected"
+            breachAlertsEmailsAllowed={
+              props.user.subscriber?.all_emails_to_primary ?? undefined === 0
```

Comment:
I think I'm confused here. Initially the `undefined === 0` threw me off, but I'm guessing that the `props.user.subscriber?.all_emails_to_primary ?? undefined` is computed first and _THEN_ the `=== 0` comparison?

But not sure if I'm seeing that behavior in a sandbox file.

```js
const props = { user: { subscriber: { all_emails_to_primary: 1 }}};
console.log(props.user.subscriber?.all_emails_to_primary ?? undefined === 0); // 1
```
```js
const props = { user: { subscriber: { all_emails_to_primary: 3 }}};
console.log(props.user.subscriber?.all_emails_to_primary ?? undefined === 0); // 3
```
But with parens to enforce nullish coalescing before strict equality, it seems to return a Boolean:
```js
const props = { user: { subscriber: { all_emails_to_primary: 3 }}};
console.log((props.user.subscriber?.all_emails_to_primary ?? undefined) === 0); // false
```
```js
const props = { user: { subscriber: { all_emails_to_primary: 0 }}};
console.log((props.user.subscriber?.all_emails_to_primary ?? undefined) === 0); // true
```
```js
const props = { user: { subscriber: null}};
console.log((props.user.subscriber.all_emails_to_primary ?? undefined) === 0); // false
```